.gitignore

```diff
@@ -0,0 +1,2 @@
+/out
+/target
```

Comment:
Followings should also be in the gitignore while using intellij. They are user specific files and will cause unnecessary conflict with other contributors.

```
# User-specific stuff
.idea/**/workspace.xml
.idea/**/tasks.xml
.idea/**/usage.statistics.xml
.idea/**/dictionaries
.idea/**/shelf
```